tests/conftest.py

```diff
@@ -87,3 +87,20 @@ def aiidalab_version(_build_config):
 @pytest.fixture(scope="session")
 def aiidalab_home_version(_build_config):
     return _build_config["AIIDALAB_HOME_VERSION"]["default"]
+
+
+@pytest.fixture(scope="function")
+def generate_aiidalab_install_output(aiidalab_exec, nb_user):
+    def _generate_aiidalab_install_output(package_name):
+        output = (
+            aiidalab_exec(f"aiidalab install --yes {package_name}", user=nb_user)
+            .decode()
+            .strip()
+        )
+
+        # Uninstall the package to make sure the test is repeatable
+        aiidalab_exec(f"aiidalab uninstall --yes --force {package_name}", user=nb_user)
```

Comment:
Hmm, I don't think this works when you pass in `package_name` as the whole url to github. Just tested this locally

```console
$ aiidalab uninstall aiidalab-widgets-base@git+https://github.com/aiidalab/aiidalab-widgets-base
...
...
$ aiidalab uninstall aiidalab-widgets-base@git+https://github.com/aiidalab/aiidalab-widgets-base
Collecting apps to uninstall... Done.

Nothing to uninstall, exiting.
```

Reply:
Just had another idea. I think it would be good to run `pip check` right after the install to ensure there are no dependency conflicts. We kind of test it I think by inspecting the output, but this would be more robust I think.

Reply:
Yes, `pip check` is a good idea, added. 

> , I don't think this works when you pass in package_name as the whole url to github. Just tested this locally

True, please check the new commit https://github.com/aiidalab/aiidalab-docker-stack/pull/376/commits/1d027eade531895b4cab598e8d93b05283a026a1, I extract the name of the app and uninstall it.